src/renderer/features/albums/components/album-detail-content.tsx

```diff
@@ -191,20 +191,22 @@ export const AlbumDetailContent = ({ tableRef }: AlbumDetailContentProps) => {
         serverId: server?.id,
     });
 
-    const carousels = [
-        {
-            data: artistQuery?.data?.items,
-            loading: artistQuery?.isLoading || artistQuery.isFetching,
-            pagination: {
-                handleNextPage: () => handleNextPage('artist'),
-                handlePreviousPage: () => handlePreviousPage('artist'),
-                hasPreviousPage: pagination.artist > 0,
-                itemsPerPage,
-            },
-            title: 'More from this artist',
-            uniqueId: 'mostPlayed',
-        },
-    ];
+    const carousels = artistQuery?.data?.items.length
```

Comment:
Thanks for the PR!

This should be changed to use the spread operator to conditionally add that single object to the array rather than conditionally rendering the entire array so that any new carousels can be added easily.

Reply:
Thanks for the feedback,

Would something like this work better?
f75daffed5b040109de88b58240af3e2d247cf3e

Reply:
Actually, add a `isHidden` property to the carousel object and then do an array filter before the `map` statement in the jsx render.

If you look at `album-artist-detail-content-tsx` you can see an example of how it was done previously.

Reply:
There's some stylistic changes I've requested if you missed the previous comment!

Reply:
I think I included all the changes in the latest commit by using array filter like used in `album-artist-detail-content.tsx`